downstream/modules/platform/proc-scm-insights.adoc

```diff
@@ -0,0 +1,25 @@
+[id="proc-scm-insights"]
+
+= SCM Type - Red Hat Insights
+
+Use the following procedure to configure playbooks to use Red Hat Insights.
+
+.Procedure
+. In the  Project *Details* page, select *Red Hat Insights* from the *SCM Type* menu.
+. Red Hat Insights requires a credential for authentication. 
```

Comment:
This does not feel like a step where you are to take action so maybe combine with the following step?